src/tests/system/tests/test_sss_override.py

```diff
@@ -15,54 +15,84 @@
+def test_sss_override__user(client: Client, provider: GenericProvider):
     """
-    :title: Locally overriding the name and POSIX attributes of a user
+    :title: Override user
     :setup:
-        1. Create POSIX user "user1"
-        2. Configure SSSD with "ldap_id_mapping = false" and start SSSD
-        3. Create local override for "user1"
-        4. Restart SSSD, this is necessary to enable local overrides
+        1. Create user
+        2. Configure SSSD with and start SSSD
+        3. Create local override and restart SSSD
     :steps:
-        1. Authenticate as "user1", then as the override name, use the short and fully qualified name
-        2. Lookup user by the overridden name, check the uid and gid
-        3. Override user's uid, gid and homedir and lookup user by both names, check the uid and gid
+        1. Login as the regular and override name
     :expectedresults:
         1. Users logins are successful
-        2. User is found and uid and gid match original values
-        3. User is found using both names and uid, gid and homedir match the new values
     :customerscenario: False
     """
-    provider.user("user1").add(
-        uid=999011, gid=999011, home="/home/user1", gecos="user", shell="/bin/bash", password="Secret123"
-    )
-
-    client.sssd.domain["ldap_id_mapping"] = "False"
+    provider.user("user1").add(password="Secret123")
     client.sssd.start()
-
     client.sss_override.user("user1").add(name="o-user1")
-
     client.sssd.restart()
 
     assert client.auth.ssh.password("user1", "Secret123"), "Failed login!"
-    assert client.auth.ssh.password(
-        f"user1@{client.sssd.default_domain}", "Secret123"
-    ), "Fully qualified name failed login!"
-
     assert client.auth.ssh.password("o-user1", "Secret123"), "Override name failed login!"
-    assert client.auth.ssh.password(
-        f"o-user1@{client.sssd.default_domain}", "Secret123"
-    ), "Override fully qualified name failed login!"
 
-    result = client.tools.getent.passwd("o-user1")
-    assert result is not None, "User not found by override name!"
-    assert result.uid == 999011, "User's uid does not match original value!"
-    assert result.gid == 999011, "User's gid does not match original value!"
 
+@pytest.mark.importance("high")
+@pytest.mark.ticket(bz=1254184)
+@pytest.mark.topology([KnownTopology.LDAP, KnownTopology.AD, KnownTopology.Samba])
+@pytest.mark.preferred_topology(KnownTopology.LDAP)
+def test_sss_override__fully_qualified_user_name(client: Client, provider: GenericProvider):
+    """
+    :title: Override using fully qualified names
+    :setup:
+        1. Create user
+        2. Configure SSSD with "use_fully_qualified_names =True", "ldap_id_mapping=False" and start SSSD
+        3. Create user override and restart SSSD
+    :steps:
+        1. Login with the username and overridden name
+        2. Login with the fully qualified username and overridden name
+    :expectedresults:
+        1. Logins fail
```

Comment:
Why this should fail? The test could use some description to explain what is going on.